app/domain/errors.rb

```diff
@@ -173,104 +178,99 @@ module AuthnIam
       module AuthnK8s
 
         CSRIsMissingSpiffeId = ::Util::TrackableErrorClass.new(
-          msg: 'CSR must contain SPIFFE ID SAN',
+          msg:  'CSR must contain SPIFFE ID SAN',
           code: "CONJ00022E"
         )
 
         PodNotFound = ::Util::TrackableErrorClass.new(
-          msg: "No pod found for '{0-pod-name}' in namespace '{1}'",
+          msg:  "No pod found for '{0-pod-name}' in namespace '{1}'",
           code: "CONJ00024E"
         )
 
         ScopeNotSupported = ::Util::TrackableErrorClass.new(
-          msg: "Resource type '{0}' is not a supported application identity. The supported resources are '{1}'",
+          msg:  "Resource type '{0}' is not a supported application identity. The supported resources are '{1}'",
           code: "CONJ00025E"
         )
 
         K8sResourceNotFound = ::Util::TrackableErrorClass.new(
-          msg: "Kubernetes {0-resource-name} {1-object-name} not found in namespace {2}",
+          msg:  "Kubernetes {0-resource-name} {1-object-name} not found in namespace {2}",
           code: "CONJ00026E"
         )
 
         CertInstallationError = ::Util::TrackableErrorClass.new(
-          msg: "Certificate could not be copied to pod: {0}",
+          msg:  "Certificate could not be copied to pod: {0}",
           code: "CONJ00027E"
         )
 
         ContainerNotFound = ::Util::TrackableErrorClass.new(
-          msg: "Container {0} was not found in the pod. Host id: {1}",
+          msg:  "Container {0} was not found in the pod. Host id: {1}",
           code: "CONJ00028E"
         )
 
         MissingClientCertificate = ::Util::TrackableErrorClass.new(
-          msg: "Client SSL certificate is missing from the header",
+          msg:  "Client SSL certificate is missing from the header",
           code: "CONJ00029E"
         )
 
         UntrustedClientCertificate = ::Util::TrackableErrorClass.new(
-          msg: "Client certificate cannot be verified by certification authority",
+          msg:  "Client certificate cannot be verified by certification authority",
           code: "CONJ00030E"
         )
 
         CommonNameDoesntMatchHost = ::Util::TrackableErrorClass.new(
-          msg: "Client certificate CN must match host name. Cert CN: {0}. " \
+          msg:  "Client certificate CN must match host name. Cert CN: {0}. " \
```

Comment:
This is cool, I was unaware you could do this.

We should do this everywhere we can to avoid the really long lines.